src/main/java/net/javadiscord/javabot/systems/commands/FormatCodeCommand.java

```diff
@@ -27,38 +26,32 @@ public ReplyCallbackAction handleSlashCommandInteraction(SlashCommandInteraction
 		var slashConfig = Bot.config.get(event.getGuild()).getSlashCommand();
 		if (idOption == null) {
 			event.getChannel().getHistory()
-				.retrievePast(10)
-				.queue(messages -> {
-					Message target = null;
-					Collections.reverse(messages);
-					for (Message m : messages) {
-						if (!m.getAuthor().isBot()) target = m;
-					}
-					if (target != null) event.getHook().sendMessageEmbeds(buildFormatCodeEmbed(target, format, slashConfig)).queue();
-					else Responses.error(event.getHook(), "Missing required arguments.").queue();
-				});
+					.retrievePast(10)
+					.queue(messages -> {
+						Message target = null;
+						Collections.reverse(messages);
+						for (Message m : messages) {
+							if (!m.getAuthor().isBot()) target = m;
+						}
+						if (target != null) {
+							event.getHook().sendMessageFormat("```%s\n%s\n```", format, StringUtils.standardSanitizer().compute(target.getContentRaw())).queue();
+						} else {
+							Responses.error(event.getHook(), "Missing required arguments.").queue();
+						}
+					});
 		} else {
 			long messageId = idOption.getAsLong();
 			event.getTextChannel().retrieveMessageById(messageId).queue(
-					m -> event.getHook().sendMessageEmbeds(buildFormatCodeEmbed(m, format, slashConfig)).queue(),
+					m -> event.getHook().sendMessageFormat("```%s\n%s\n```", format, StringUtils.standardSanitizer().compute(m.getContentRaw())).queue(),
```

Comment:
[`allowedMentions`](https://javadoc.io/doc/net.dv8tion/JDA/latest/net/dv8tion/jda/api/utils/AllowedMentions.html#allowedMentions(java.util.Collection)) should also be set for further security against mentions not recognized by the sanitizer.